docs/usage.rst

```diff
@@ -95,19 +95,23 @@ To validate a RAML file with Python:
 
    >>> from ramlfications import validate
    >>> RAML_FILE = "/path/to/my-api.raml"
-   >>> validate(RAML_FILE)
-   >>>
+   >>> errors = validate(RAML_FILE)
+   >>> if not errors:
+   >>>     print("Success!")
 
 .. code-block:: python
 
    >>> from ramlfications import validate
    >>> RAML_FILE = "/path/to/invalid/no-title.raml"
-   >>> validate(RAML_FILE)
-   InvalidRootNodeError: RAML File does not define an API title.
+   >>> errors = validate(RAML_FILE)
```

Comment:
Yeah I think that may be the approach here, have something like `MultipleValidationErrors(Exception)`.